docs/user-guide/api-mediation/configuration-customizing-java-heap-sizes.md

```diff
@@ -0,0 +1,28 @@
+# Customizing Java Heap sizes
+
+:::info Role: system programmer
+::: 
+
+As a systems programmer, you can customize the Java heap memory size available for the API Mediation Layer components.
```

Comment:
Perhaps it would be useful to describe what "heap memory size" is...

Reply:
Ok, I'll give it a try